test/Cache-test.js

```diff
@@ -177,5 +178,20 @@ describe('raptor-cache' , function() {
             ],
             done);
     });
-});
 
+    // Promise tests that will only run on Node 4+
+    if (promisesExist) {
```

Comment:
Yes, we can drop support for Node < 4, but to be safe we should increment the major version